### DISCUSSION TITLE
timetable.execution_log

### DISCUSSION THREAD

**Discussion by petr-klinger** [cybertec-postgresql/pg_timetable#441] [General]
The "timetable.execution log" table is not empty for running applications. Therefore, change # 436 is needed instead of the current SQL statement "ALTER TABLE timetable.execution_log ADD COLUMN txid INTEGER NOT NULL;" use SQL statement "ALTER TABLE timetable.execution_log ADD COLUMN txid SERIAL NOT NULL;"

**Comment by pashagolub** [comment]
oh, I see your point.

**Comment by pashagolub** [comment]
I would prefer 

```
ALTER TABLE timetable.execution_log
    ADD COLUMN txid INTEGER NOT NULL DEFAULT 0;
```
